tools/mixmodel4repeated_measures/mixmodel.xml

```diff
@@ -0,0 +1,268 @@
+<tool id="mixmodel" name="mixmodel" version="3.1.0">
+    <description>ANOVA for repeated measures statistics</description>
+
+    <requirements>
+        <requirement type="package">r-lme4</requirement>
+        <requirement type="package" version="3.1_3">r-lmertest</requirement>
+        <requirement type="package" version="3.2_0">r-ggplot2</requirement>
+        <requirement type="package" version="2.3">r-gridExtra</requirement>
+        <requirement type="package" version="2.40.0">bioconductor-multtest</requirement>
+    </requirements>
+
+    <stdio>
+        <exit_code range="1:" level="fatal" />
+    </stdio>
+
+    <command><![CDATA[

```

Comment:
maybe you can use here `detect_errors` https://docs.galaxyproject.org/en/latest/dev/schema.html#error-detection 
and remove the stdio section

Reply:
I have no idea what the stdio section handles in the first place (appart that it is about error status?) so I am not sure what value of the detect_errors attribute would be equivalent to the current stdio parameters...

Reply:
`<exit_code range="1:" level="fatal" />`
tells galaxy to consider this tool is in error if the error code is in range 1 - 255.
So the only case when this tool in not in error is when it terminates with status code 0.

In R, we can stop a program and provide the error code with:
```
base::quit(status = 0) ## exits without error
base::quit(status = 25) ## exits with error code 25.
```
This status should be unique per error, so we can easily tell what kind of error occurred just by looking for the error code.

Reply:
Thank you for the complete explanation! Now I even know how to relevantly use it with R (for futur works)

Reply:
Adressed in 367a4f4